src/ui/components/UserReview/index.js

```diff
@@ -0,0 +1,54 @@
+/* @flow */
+import * as React from 'react';
+
+import { nl2br, sanitizeHTML } from 'core/utils';
+import LoadingText from 'ui/components/LoadingText';
+import UserRating from 'ui/components/UserRating';
+import type { UserReviewType } from 'amo/actions/reviews';
+
+import './styles.scss';
+
+type Props = {|
+  byLine: React.Node | null,
+  children?: React.Node,
+  isReply?: boolean,
+  review: ?UserReviewType,
+|};
+
+const UserReview: React.ComponentType<Props> = ({
+  byLine,
+  children,
+  review,
+  isReply = false,
+}: Props) => {
+  let body = (
+    <p className="UserReview-body">
+      <LoadingText />
+    </p>
+  );
+
+  if (review) {
+    body = (
+      <p
+        className="UserReview-body"
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={sanitizeHTML(nl2br(review.body), ['br'])}
+      />
+    );
+  }
+
+  return (
+    <div className="UserReview">
+      {body}
+      <div className="UserReview-byLine">
+        {review && !isReply ? (
```

Comment:
Since `isReply` no longer makes sense in this context, I think it would be better to name it more explicitly like `showRating` or something.